packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.visualtest.ts

```diff
@@ -0,0 +1,17 @@
+import { Casper, IPhantomCSS } from '../../visualtest/PhantomCssInterface';
+import { baseUrl } from '../../common/VisualTest';
+declare var phantomcss: IPhantomCSS;
+declare var casper: Casper;
+/* tslint:disable:no-function-expression */
+casper.start(baseUrl + 'breadcrumb').
+  then(function () {
+    phantomcss.screenshot('.Breadcrumb', 'Breadcrumb_not_pressed');
+  }).then(function () {
+    this.mouse.move('.Breadcrumb');
+    phantomcss.screenshot('.Breadcrumb', 'Breadcrumb_hovered');
```

Comment:
As png is captured with component id, hovering over a specific part and taking a png may not be possible.